### DISCUSSION TITLE
webgl samples

### DISCUSSION THREAD

**Discussion by unicomp21** [google/filament#6934] [Q&A]
the webgl samples run fine on macos/intel/safari, but all other browsers (on macos/intel) lock up when running the webgl samples?  anyone else seeing this?

**Comment by unicomp21** [comment]
@romainguy i tried chrome, edge, brave.  all would appear to lock up, while safari runs fine.

**Reply by romainguy** [reply]
Chromium has an issue that causes shaders to take a long time to compile. Recent changes made this worse but we fixed it although I don't think we've updated the samples. @bejado

**Reply by unicomp21** [reply]
on windows 11, all cores become cpu bound (chrome, edge, brave)

**Reply by romainguy** [reply]
See my previous reply.

**Reply by unicomp21** [reply]
@romainguy how many minutes is a "long time"?  From what I can tell, the compile never ends?  @bejado?

**Reply by bejado** [reply]
The compiler *does* eventually finish, but on my MacBook Pro it took around 1-2 minutes to render a glTF.

**Reply by bejado** [reply]
> Chromium has an issue that causes shaders to take a long time to compile. Recent changes made this worse but we fixed it although I don't think we've updated the samples. @bejado 

npm has been updated, but correct I don't think samples have been.
